src/server/extensions/compatibility-layer/searx/bridge/shims.py

```
@@ -0,0 +1,283 @@
+        EngineTraits=EngineTraits,
+        EngineTraitsMap=dict,
+    )
+    setattr(enginelib, "traits", traits)
+    return enginelib
+
+
+def _network():
+    network = _module("searx.network")
+    for verb in HTTP_VERBS:
+        setattr(network, verb, functools.partial(rpc.fetch, verb.upper()))
+    setattr(network, "request", lambda method, url, **kwargs: rpc.fetch(str(method).upper(), url, **kwargs))
```

Ruff (0.16.0)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)